task_4_3.py

```diff
@@ -0,0 +1,37 @@
+import aiohttp
+import asyncio
+import time
+from task_4_1 import urls
+import os
+import aiofiles
+
+
+async def download_as(url):
+    folder = 'data_asyncio'
+    if not os.path.exists(folder):
+        os.mkdir(folder)
+    async with aiohttp.ClientSession() as session:
+        async with session.get(url) as responce:
+            extention = responce.headers.get('Content-Type').split('/')[1]
+            url = url.split('?')
+            url = url[0].split('/')
+            url = url[-1].replace('.jpeg','')
+            async with aiofiles.open(f'{folder}/{url}.{extention}','wb') as f:
```

Comment:
А тут вы почему-то поменяли способ. Только зачем?
```suggestion
            url, *_ = os.path.basename(url).split('?')
            async with aiofiles.open(fos.path.join(folder, url),'wb') as f:
```